models/taxon.py

```diff
@@ -0,0 +1,41 @@
+import sqlalchemy as sql
+
+class Taxon():
+    @staticmethod
+    def search_by_name(db_conn, term, page=1, per_page=10):
```

Comment:
This does not necessarily need to be in a class, but I'd like to experiment with code structure a bit. I'd like to try creating ORM classes for database entities to make individual queries simpler (<https://docs.sqlalchemy.org/en/20/orm/quickstart.html>)